course4/lecture4_en.md

```diff
@@ -0,0 +1,306 @@
+# Modern Programming Concepts: Tuples, Structures, and Enumerated Types
+
+Hello everyone, welcome to the open course on modern programming concepts brought to you by the IDEA's Digital Infrastructure Innovation. Today's topic is tuples, structures, and enumerated types. In a nutshell, we will be exploring how to customize data structures in MoonBit.
+
+# Review: Tuples
```

Comment:
Maybe use a second level heading here and for everything else? There should be only one H1 in one document.